app/dashboard/templates/profiles/profile.html

```diff
@@ -234,15 +234,65 @@
                       </div>
                     </div>
                   </div>
-                </div>
-                {% endif %}
+                </div> {% endif %}
               </div>
             {% endfor %}
-          </div>
-        </div>
+		{% else %}
+		<br>
```

Comment:
Do we need a break ? Can we use `mt-2` to the below `div` ?

Reply:
we could use a mt-2. I just saw a BR somewhere further above, hence I tried it that way, but, sure, we can go with margins if you prefer them generally.